lib/matplotlib/legend.py

```diff
@@ -857,10 +857,10 @@ def get_texts(self):
         r"""Return the list of `~.text.Text`\s in the legend."""
         return silent_list('Text', self.texts)
 
-    def set_title(self, title, prop=None):
+    def set_title(self, title, prop=None, *, align=None):
         """
-        Set the legend title. Fontproperties can be optionally set
-        with *prop* parameter.
+        Set the legend title. Alignment can be set with *align* parameter.
+        Fontproperties can be optionally set with *prop* parameter.
```

Comment:
It would be better if you could re-write these as [`numpydoc`](https://numpydoc.readthedocs.io/en/latest/format.html).